modules/editor-events/module.php

```diff
@@ -20,6 +22,13 @@ public static function get_editor_events_config() {
 
 		$settings = [
 			'can_send_events' => $can_send_events,
+			'elementor_version' => ELEMENTOR_VERSION,
+			'site_url' => get_site_url(),
+			'wp_version' => get_bloginfo( 'version' ),
+			'user_agent' => esc_html( Utils::get_super_global_value( $_SERVER, 'HTTP_USER_AGENT' ) ),
+			'site_language' => get_locale(),
```

Comment:
```suggestion
			'site_language' => get_bloginfo( 'language' ),
```
Which one is better?

Reply:
Justa. question, get_locale() might be better :)

Reply:
I checked what we are already using for System Info page and just re-used most of functions for this. The main difference is: 
`get_bloginfo( 'language' )` - will return 'en-US'
`get_locale()` - will return `en_US`

not a big difference actually :)